### ISSUE TITLE
HomePage.react.js eslint errors

### ISSUE THREAD

**Issue by mxstbr** [mxstbr/react-boilerplate#23]
eslint current shows the following errors in `HomePage.react.js` in Atom:

```
Expected indentation of 2 space characters but found 0.
```

for lines 13, 14 and 15. Which doesn't make any sense, they aren't indented differently from the rest. If you change it to two spaces, the errors go away, but 4 are needed for it to look the way it should. No idea why that happens...

**Comment by mxstbr** [comment]
Could be related to https://github.com/eslint/eslint/issues/4372, if it is eslint@2.0.0-alpha-1 fixes the issue. Taking a look...

**Comment by mxstbr** [comment]
I've fixed it by updating to eslint 2.0, but that release is currently in alpha status. I've pushed the `eslint2` branch which has the updated `.eslintrc` (the structure has changed in a breaking way) and there are no eslint errors at all.

I don't feel comfortable forcing everybody to globally install a package in alpha status, so I'll leave the errors for now, leave the issue open and wait for the eslint 2.0 release.
